src/applications/gi/components/Checkbox.jsx

```diff
@@ -73,7 +73,6 @@ class Checkbox extends React.Component {
     return (
       <div className={className}>
         <input
-          autoComplete="false"
```

Comment:
This `Checkbox` component is also used here (https://github.com/department-of-veterans-affairs/vets-website/blob/16091-remove-checkbox-autocomplete/src/applications/gi/components/search/InstitutionFilterForm.jsx#L4) but it should be safe to remove this attribute, especially given the fact that having this attribute on the checkbox makes it unusable by assistive devices.